pagination/pagetoken_test.go

```diff
@@ -35,6 +36,52 @@ func TestParseOffsetPageToken(t *testing.T) {
 		assert.Equal(t, int64(30), pageToken3.Offset)
 	})
 
+	t.Run("skip", func(t *testing.T) {
+		request1 := &freightv1.ListSitesRequest{
+			Parent:    "shippers/1",
+			PageSize:  20,
+			PageToken: "",
+			Skip:      30,
+		}
+		pageToken1, err := ParsePageToken(request1)
```

Comment:
@stenehall to improve the reporting on these test cases, you could wrap each of them in their own `t.Run` with a unique description of the test case

Reply:
@stenehall nice! FYI you can nest t.Run - so if you do

```go
t.Run("skip", func(t *testing.T) {
  t.Run("handle empty token with skip", func(t *testing.T) {
    // ...
  })
})
```

the reporting will get flattened to `skip: handle empty token with skip`